src/components/App.js

```diff
@@ -1,32 +1,33 @@
-import PropTypes from 'prop-types';
+import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 import Calculate from '../Logic/calculate';
 
-export default function App(props) {
-  const {
-    total, next, operation, buttonName,
-  } = props;
-  const obj = { total, next, operation };
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
 
-  return (
-    <>
-      <div className="display-container"><Display /></div>
```

Comment:
- [ ] <b>REQUIRED:</b> Your app is only displaying the first number, after I click on the operator,  the second number is being displayed. kindly fix that.